test/meta_merge/core_test.cljc

```diff
@@ -1,6 +1,17 @@
 (ns meta-merge.core-test
-  (:require [clojure.test :refer :all]
-            [meta-merge.core :refer :all]))
+  (:require #?(:clj  [clojure.test :refer :all]
+               :cljs [cljs.test :refer-macros [deftest is testing]])
+            [meta-merge.core :refer [meta-merge metadatable?]]))
+
+(def metable? (or metadatable? #'meta-merge.core/metadatable?))
+(deftest test-metadatable
```

Comment:
Okay, no problem. I added the tests as part of my debugging and they seemed to be quite useful to diagnose potential problems. Easily removed.